script.js

```diff
@@ -93,13 +113,26 @@ function changeElms(){
 
 // TODO: select and change the span tags
 //       then loop through the elements and change them.
+const y= document.querySelectorAll("span");
+y[0] .style.backgroundColor="pink";
+y[1].style.backgroundColor="black";
+y[2].style.backgroundColor="white";
+y[3].style.backgroundColor="red";
+y[4].style.backgroundColor="white";
+y[5].style.backgroundColor="yellow";
+y[6].style.backgroundColor="black";
+y[7].style.backgroundColor="blue";
+y[8].style.backgroundColor="white";
+y[9].style.backgroundColor="red";
+
 
 
 // TODO: Go back to index.html and add
 //       a class called "jump" to 10 elements.
 //       These can be any elements you want.
 //       For example: 2 h1 tags, 3 p tags, 5 span tags.
-
+const f= document.querySelectorAll("jump");
```

Comment:
`jump` isn't a tag. For `p` and `span`, those are tags. So we can use `"p"` and `"span"`. But, `jump` is a class. Remember, we use `#` for ids and `.` for classes. Check out: https://flukeout.github.io/